org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLFormatter.java

```diff
@@ -103,6 +103,14 @@ private void format(Node node, int level, int end, XMLBuilder xml) {
 				if (level == 0) {
 					xml.linefeed();
 				}
+			}

```

Comment:
Please keep my original code if you ar OK with my constants PROCESSING_INSTRUCTION_NODE. In otherwise process prolog before processing instruction.